cluster/addons/fluentd-gcp/fluentd-gcp-image/google-fluentd-journal.conf

```diff
@@ -63,7 +63,17 @@
 <match reform.**>
   type record_reformer
   enable_ruby true
-  tag kubernetes.${tag_suffix[4].split('-')[0..-2].join('-')}
+  tag raw.kubernetes.${tag_suffix[4].split('-')[0..-2].join('-')}
+</match>
+
+# Detect exceptions in the log output and forward them as one log entry.
+<match raw.kubernetes.**>
+  type detect_exceptions
+  remove_tag_prefix raw
+  message log
+  multiline_flush_interval 5
+  max_bytes 50000
```

Comment:
Based on our recent discussion, I've changed the limit back to 50k, since we cannot know measure the overhead for the message in addition to the actual payload.